### PR TITLE
update: Add underline for header links

### PR DESCRIPTION
**Pull request by gabestein**

### CODE REVIEW

File: src/styles/base.scss [hunk 1]
```diff
@@ -34,8 +34,11 @@
     h5,
     h6 {
         a {
-            text-decoration: none;
+            text-decoration: underline;
             color: inherit;
+            &:hover, &:visited {
```
Comment (idreyn) [inline src/styles/base.scss]: IMO we should continue to show the underline even after the link has been visited. We might need to move `color: inherit` into this block, though.

Comment (gabestein) [reply src/styles/base.scss]: Ah, good point. Will do both.

**Review by idreyn — APPROVED** [review]
🤘

**Comment by idreyn** [comment]
Ah, oof. Just realized that this PR is in `pubpub-editor`, which Travis recently merged into `pubpub` itself. You'll want to make the equivalent changes to `client/components/Editor` over there.

**Pull request closed without merging**